### PR TITLE
[Term Entry] Kotlin Strings .padEnd()

### PR DESCRIPTION
**Pull request by Fejiro001**
### Description

- Added the .padEnd() method for Kotlin Strings
- Closes Issue #3728  

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/kotlin/concepts/strings/terms/padEnd/padEnd.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+```kotlin
+fun main() {
+  val padWithSpace = "Hello".padEnd(8)
+  println("'$padWithSpace'")
+
+  val padWithCharacter = "Hello".padEnd(8, '!')
+  println("'$padWithCharacter'")
+
+// string is returned when its length is greater than the specified string
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/strings/terms/padEnd/padEnd.md:42]:
```suggestion
```
No need for comment here

File: content/kotlin/concepts/strings/terms/padEnd/padEnd.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+---
+Title: '.padEnd()'
+Description: 'Pads a String using Characters'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Strings'
+  - 'Methods'
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/strings/terms/padEnd/padEnd.md:9]:
```suggestion
  - 'Methods'
  - 'Kotlin'
```

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
2 changes from my side

**Comment by dakshdeepHERE** [comment]
Hey @Fejiro001 👋, this looks good for 2nd review. 🚀

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Fejiro001 I did a second review and left a few comments.

**Review by SSwiniarski — APPROVED** [review]
@Fejiro001 Did one minor edit and this is ready to merge.

**Pull request merged**